### PR TITLE
www-misc/profile-sync-daemon: bring back version 5.75

### PR DESCRIPTION
**Pull request by AnAverageHuman**
This commit partially reverts 79f41a54cc55c39e0a4ac9fba8115469f01d262e.
Bug: https://bugs.gentoo.org/678658

### CODE REVIEW

**Comment by juippis** [comment]
By modern Gentoo standards, USE flags shouldn't be used to control installation of small files, and you should just install them unconditionally. So you should at least remove zsh-completion USE flag, and if that systemd USE is only used to control installation of service files, like it seems to be, you should remove that too and install them unconditionally.

**Comment by AnAverageHuman** [comment]
Removed the `zsh-completion` USE flag, but the script seems to prioritize systemd over openrc detection, which is problematic. I've fixed the `usex` command to only install one or the other.

File: www-misc/profile-sync-daemon/profile-sync-daemon-5.75.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+src_install() {
+	emake DESTDIR="${ED}" COMPRESS_MAN=0 \
+		"$(usex systemd "install-systemd-all" "install-openrc-all")"
+
+	fperms -x /etc/cron.hourly/psd-update
+}
+
+pkg_postinst() {
+	elog "The cronjob is -x by default. You might want to set it +x if you"
```
Comment (mgorny) [inline www-misc/profile-sync-daemon/profile-sync-daemon-5.75.ebuild]: Why would you use systemd timer with USE=-systemd? ;-)

**Comment by mgorny** [comment]
Please add yourself as proxy-maint.

**Comment by AnAverageHuman** [comment]
Done, but I will not be able to test versions newer than 5.75 as that is the last version that does not require systemd.

**Comment by mgorny** [comment]
I suppose that's fine as long as you take care of bumping anyway, and at least build-test them (AFAICS you can do `--buildonly` without systemd installed).

**Pull request closed without merging**